src/components/containers/Panel.js

```diff
@@ -106,7 +108,7 @@ class Panel extends Component {
     );
 
     return (
-      <div className={bem('panel')}>
+      <div className={`panel${tabs ? ' panel--with-tabs' : ''}`}>
```

Comment:
yeah, I think we're not really using that bem stuff, we should probably remove it at some point..

Reply:
This was added by @aulneau so we should get his input... we should either do one big pass where we remove it or add it everywhere in a single PR but agreed that we should standardize!

Reply:
no : ) this was added before aulneau, we've just all been adding to it, but yeah @aulneau what do you think about that bem utility, useful?

Reply:
Oh, did you mean the `bem()` function or the actual BEM pattern?

And yes, the particular edit you're commenting on came from Thomas ;)

Reply:
actually come to think of it I was going to refactor it a bit, good that you chose this one to comment on!

Reply:
in the same family of functions as things like `bem()` I like https://github.com/JedWatson/classnames FWIW...

Reply:
I meant the `bem` function, which was introduced before @aulneau started contributing to this project : )